packages/components/src/locales/en-us/messages-word-list.json

```diff
@@ -139,5 +139,10 @@
         "message": "with filtering",
         "description": "Download list filtering check",
         "component": "WordLanguagePanel"
+    },
+    "WORDLIST_DOWNLOAD_QUIZLET_CHECK": {
+        "message": "for quizlet",
+        "description": "Download list for quizlet",
+        "component": "WordLanguagePanel"
```

Comment:
let's make it more generic, because hopefully it will apply to more than just quizlet 

```
 "WORDLIST_DOWNLOAD_FLASHCARD_CHECK": {
        "message": "for flashcards",
        "description": "Download list for flashcards",
        "component": "WordLanguagePanel"
}
```